release.ini

```diff
@@ -24,7 +22,7 @@ blog.password =
 
 [server:main]
 use = egg:waitress#main
-host = 127.0.0.1
+host = 0.0.0.0
```

Comment:
don't replace development.ini with release.ini. Both should exist, and development.ini should be unchanged.

Anything actually hosting this app will be set to use release.ini. For our development purposes, we will use development.ini